recipe/conda_build_config.yaml

```diff
@@ -12,13 +12,24 @@ blas_impl:
   - generic
 
 github_actions_labels:
-- cirun-openstack-gpu-large    # [x86 or x86_64]
+- cirun-openstack-gpu-4xlarge    # [x86 or x86_64]
 - cirun-openstack-cpu-large
 
 megabuild:
 - true      # [linux]
 - false     # [osx]
+- true      # [win]
 
 zip_keys:
   - - github_actions_labels
     - blas_impl
+
+c_compiler:        # [win]
+- vs2019           # [win]
+cxx_compiler:      # [win]
+- vs2019           # [win]
+
+c_stdlib:          # [linux64]
+  - sysroot        # [linux64]
+c_stdlib_version:  # [linux64]
+  - 2.17           # [linux64]
```

Comment:
All of this is redundant and should be removed (just leads to outdated cruft lying around in the CBC)